ignite/cmd/plugin_default.go

```diff
@@ -17,7 +17,7 @@ type defaultPlugin struct {
 
 const (
 	PluginNetworkVersion = "v0.2.0"
-	PluginNetworkPath    = "github.com/ignite/cli-plugin-network@" + PluginNetworkVersion
+	PluginNetworkPath    = "github.com/ignite/cli/v28-plugin-network@" + PluginNetworkVersion
```

Comment:
This one was right before the replace:

```suggestion
	PluginNetworkPath    = "github.com/ignite/cli-plugin-network@" + PluginNetworkVersion
```